### ISSUE TITLE
"python <defunct>" jobs when multiple files are sent to pyas2

### ISSUE THREAD

**Issue by silverfox87ls** [abhishek-ram/pyas2#50]
When pyas2 picks up one file at a time I have no problems. When multiple files are put in the outbound folder it sometimes doesn't process them until I "touch" all the files to update the timestamp. When it does process them as a batch I'm left with a "python <defunct>" job for each file. After several defunct jobs build up I have to restart pyas2 for it to function automatically again.
Python 2.7 on Oracle Linux 6.10

**Comment by abhishek-ram** [comment]
Yes even I have found this functionality to be unreliable sometimes. I will see if I can put in a fix but the best solution would be to call the sendmessage function by the process writing the files to the directory.

**Comment by silverfox87ls** [comment]
Scripting the sendmessage command helps, thank you.
